client/js/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -70,181 +70,167 @@ export default {
     },
   },
   computed: {
-    /**
-     * Returns list of objects representing the available moves,
-     * based on the selected card
-     */
-    moveChoices() {
-      if (!this.selectedCard) return [];
-      let res = [];
-      let cardRank;
-      if (this.resolvingSeven) {
-        if (!this.cardSelectedFromDeck) return [];
-        cardRank = this.cardSelectedFromDeck.rank;
-      } else {
-        if (!this.selectedCard) return [];
-        cardRank = this.selectedCard.rank;
-      }
-      // Re-usable move-objects
-      const allMovesAreDisabled = !this.isPlayersTurn || this.frozenId === this.selectedCard.id;
-      const allMovesDisabledExplanation = !this.isPlayersTurn
-        ? "It's not your turn"
-        : 'This card is frozen';
-
-      let disabledText;
-      if (allMovesAreDisabled) {
-        disabledText = allMovesDisabledExplanation;
+    // Determines if any moves are available
+    allMovesAreDisabled() {
+      return !this.isPlayersTurn || this.frozenId === this.selectedCard.id;
+    },
+    // Determines which disabled text to display
+    disabledText() {
+      if (this.allMovesAreDisabled) {
+        return !this.isPlayersTurn ? "It's not your turn" : 'This card is frozen';
       }
-
-      // Points
-      const pointsDescription = `Gain ${cardRank} point` + (cardRank === 1 ? '' : 's');
-      const pointsMove = {
+      return '';
+    },
+    pointsMove() {
+      const pointsDescription =
+        `Gain ${this.selectedCard.rank} point` + (this.selectedCard.rank === 1 ? '' : 's');
+      return {
         displayName: 'Points',
         eventName: 'points',
         moveDescription: pointsDescription,
-        disabled: allMovesAreDisabled,
-        disabledExplanation: disabledText,
+        disabled: this.allMovesAreDisabled,
+        disabledExplanation: this.disabledText,
       };
-      // Scuttling
+    },
+    scuttleMove() {
       const scuttleDisabled = !this.isPlayersTurn || !this.hasValidScuttleTarget;
```

Comment:
Let's clean this up while we're in here. 
```suggestion
      const scuttleDisabled = !this.allMovesAreDisabled || !this.hasValidScuttleTarget;
```

Reply:
This change is causing several tests to fail from basicMoves, oneOffs, sevens. Here's an example from basicMoves:

`    // Player scuttles 6 of diamonds with 7 of clubs
    cy.get('[data-player-hand-card=7-0]').click(); // 7 of clubs
    cy.get('[data-move-choice=scuttle]').click();
    cy.get('[data-opponent-point-card=6-2]').click(); // 6 of hearts
    assertGameState(0, {
      p0Hand: [Card.ACE_OF_CLUBS, Card.ACE_OF_SPADES],
      p0Points: [Card.TWO_OF_CLUBS, Card.TEN_OF_HEARTS],
      p0FaceCards: [Card.KING_OF_SPADES],
      p1Hand: [Card.TEN_OF_CLUBS, Card.TEN_OF_SPADES],
      p1Points: [Card.ACE_OF_DIAMONDS],
      p1FaceCards: [Card.QUEEN_OF_HEARTS],
      scrap: [Card.SEVEN_OF_CLUBS, Card.SIX_OF_HEARTS],
    });
    cy.log('Successfully scuttled as p0');`

The scuttle move choice on the 7 was disabled with the 'You can only scuttle smaller point cards' disabled text. The move should be legal of course. Im not sure what the logic here is.

Reply:
The scuttle move choice on the 7 was disabled with the 'You can only scuttle smaller point cards' disabled text. The move should be legal of course. Im not sure what the logic here is.

Reply:
Oops I included a ! That shouldn't be there

Reply:
it should be
```suggestion
      const scuttleDisabled = this.allMovesAreDisabled;
```